keel-ec2-plugin/keel-ec2-plugin.gradle.kts

```diff
@@ -17,6 +17,7 @@ dependencies {
   implementation("org.springframework.boot:spring-boot-autoconfigure")
   implementation("com.netflix.frigga:frigga")
   implementation("org.jetbrains.kotlinx:kotlinx-coroutines-slf4j")
+  implementation("io.swagger.core.v3:swagger-annotations:2.1.2")
```

Comment:
Not something we'd  want to bring in for `keel-api` but (at the moment at least) the models in this module are not exposed to plugins.